src/App.js

```diff
@@ -1,24 +1,64 @@
 import React from 'react';
 import './App.scss';
 
-const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+class App extends React.Component {
+  state = {
+    word: '',
+    goodsFromServer: [
+      'Dumplings',
+      'Carrot',
+      'Eggs',
+      'Ice cream',
+      'Apple',
+      'Bread',
+      'Fish',
+      'Honey',
+      'Jam',
+      'Garlic',
+    ],
+  }
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+  addGoods = (item) => {
+    this.setState({ word: `${item}` });
+  };
+
+  render() {
+    const { word, goodsFromServer } = this.state;
+
+    return (
+      <div className="App">
+        <h1 className="title">
+          Selected good:
+          {` ${word}`}
+        </h1>
+        <button
+          type="button"
+          className="delete-btn"
+          onClick={() => {
+            this.addGoods('');
```

Comment:
create method resetSelectedGood, you don't add any good, when clicking on this button